tests/unit/versioning/VersionID.js

```diff
@@ -39,4 +39,52 @@ describe('test generating versionIds', () => {
         assert.strictEqual(vids.length, count);
         assert.deepStrictEqual(vids, decoded);
     });
+
+    it('simple tiny version test', () => {
+        const vid = '98376906954349999999RG001  145.20.5';
+        const tinyVersionId = VID.tinyEncode(vid);
+        const orig = VID.tinyDecode(tinyVersionId);
+        assert.strictEqual(vid, orig);
```

Comment:
I am generally pretty skeptical of using two methods to check equality here. This does not guarantee they are right only that they are the same. Had a production failure once because our JSON tool had an issue that both the reader and writer had and the only test case was like this. This may not apply here as I have no written any complicated encoding myself but just my two cents.

Reply:
There is a test below which tests a bunch of random strings.

Reply:
Right, that takes a bunch of random strings, encodes them, and then decodes them back for assertions.
But that middle part is not tested at all, again this may not be something that is generally tested in these scenarios I don't know. But there are no one-way tests like:

```javascript
assert.strictEqual(Vid.tinyEncode('some random string'), 'some constant encoded result');
```

Then of course a separate one way test the other way.

Reply:
ok